apps/vr-tests-react-components/.storybook/preview.js

```diff
@@ -67,3 +68,5 @@ setAddon({
 
 /** @type {import("@fluentui/react-storybook-addon").FluentParameters} */
 export const parameters = { layout: 'none', mode: 'vr-test' };
+
+export const decorators = [withFluentProvider];
```

Comment:
Why this was added: https://github.com/microsoft/fluentui/pull/26504#issuecomment-1405366675